<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Copyright (C) 2009 Pascal Mues</author>
 <class>OpenImgStackDialog</class>
 <widget class="QDialog" name="OpenImgStackDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bilderstack-Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1" colspan="4">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Dateiname</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="uiSelectFile">
       <property name="text">
        <string>Auswählen</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Verzeichnis</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Dateiendung</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Start Index</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="3">
      <widget class="QCheckBox" name="uicbZeroFill">
       <property name="text">
        <string>Index mit Nullen auffüllen</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="uileFileName"/>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="uileFilePath"/>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="uisbIndexMin">
       <property name="maximum">
        <number>511</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="uileExtension"/>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="uisbWidth">
       <property name="maximum">
        <number>512</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="uisbHeight">
       <property name="maximum">
        <number>512</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Bild Breite</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Bild Höhe</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>End Index</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="uisbIndexMax">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>512</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Index Länge</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="uisbIndexLength">
       <property name="maximum">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="uibtnCancel">
     <property name="text">
      <string>Abbrechen</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="uibtnOkay">
     <property name="text">
      <string>Okay</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>uileFileName</tabstop>
  <tabstop>uiSelectFile</tabstop>
  <tabstop>uileFilePath</tabstop>
  <tabstop>uileExtension</tabstop>
  <tabstop>uisbWidth</tabstop>
  <tabstop>uisbHeight</tabstop>
  <tabstop>uisbIndexMin</tabstop>
  <tabstop>uisbIndexMax</tabstop>
  <tabstop>uisbIndexLength</tabstop>
  <tabstop>uicbZeroFill</tabstop>
  <tabstop>uibtnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
